cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -252,10 +252,9 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
                          @QueryParam("orderBy") SortOrder orderBy,
                          @QueryParam("nameFilter") String nameFilter,
                          @QueryParam("nameFilterType") NameFilterType nameFilterType,
-                         @QueryParam("latestOnly") Boolean latestOnly,
+                         @QueryParam("latestOnly") @DefaultValue("true") Boolean latestOnly,
                          @QueryParam("sortCreationTime") Boolean sortCreationTime
-      )
-      throws Exception {
+  ) throws Exception {
```

Comment:
nit: `)` should go on the previous line to be consistent with rest of the project

Reply:
updated